core/src/components/header/test/condense/header.e2e.ts

```diff
@@ -7,24 +7,37 @@ test.describe('header: condense', () => {
     skip.rtl();
 
     await page.goto('/src/components/header/test/condense');
-    const header = page.locator('#collapsibleHeader');
+    const largeTitleHeader = page.locator('#largeTitleHeader');
+    const smallTitleHeader = page.locator('#smallTitleHeader');
     const content = page.locator('ion-content');
 
-    await expect(header).toHaveAttribute('aria-hidden', 'true');
+    await expect(smallTitleHeader).toHaveAttribute('aria-hidden', 'true');
+
+    expect(await largeTitleHeader.screenshot()).toMatchSnapshot(
+      `header-condense-large-title-initial-diff-${page.getSnapshotSettings()}.png`
+    );
 
     await content.evaluate((el: HTMLIonContentElement) => el.scrollToBottom());
     await page.waitForChanges();
 
+    expect(await smallTitleHeader.screenshot()).toMatchSnapshot(
+      `header-condense-large-title-collapsed-diff-${page.getSnapshotSettings()}.png`
+    );
```

Comment:
The screenshots here seem to be in different states of the condensing animation. You may need to use `page.waitForTimeout()` to wait a little longer than `waitForChanges` does.

Reply:
The only downside to `page.waitForTimeout` is it will likely become flaky in future runs. Is there another selector that we can rely on, that informs the DOM that the animation is complete/screenshot is ready to be captured? 

Then we could do something like:

```ts
await page.waitForSelector('.some-class');
```

& it wouldn't be impacted by if the Playwright test runs slow.